string state_request

---

string state_answer

# Target info
int8[] target_id
geometry_msgs/Point[] target_pose
